ios/MusicControlManager.m

```diff
@@ -9,21 +9,56 @@
 @interface MusicControlManager ()
 
 @property (nonatomic, copy) NSString *artworkUrl;
+@property (nonatomic, strong) NSDictionary *mediaKeys;
 
 @end
 
+NSString * const MEDIA_PLAYBACK_RATE = @"playbackRate";
+
 @implementation MusicControlManager
 
 @synthesize bridge = _bridge;
 
 RCT_EXPORT_MODULE()
 
+- (NSDictionary *) mediaKeys {
+    if (_mediaKeys == nil) {
+        _mediaKeys = @{
+                @"album": MPMediaItemPropertyAlbumTitle,
+                @"trackCount": MPMediaItemPropertyAlbumTrackCount,
+                @"trackNumber": MPMediaItemPropertyAlbumTrackNumber,
+                @"artist": MPMediaItemPropertyArtist,
+                @"composer": MPMediaItemPropertyComposer,
+                @"discCount": MPMediaItemPropertyDiscCount,
+                @"discNumber": MPMediaItemPropertyDiscNumber,
+                @"genre": MPMediaItemPropertyGenre,
+                @"persistentID": MPMediaItemPropertyPersistentID,
+                @"duration": MPMediaItemPropertyPlaybackDuration,
+                @"title": MPMediaItemPropertyTitle,
+                @"elapsedPlaybackTime": MPNowPlayingInfoPropertyElapsedPlaybackTime,
+                @"playbackRate": MPNowPlayingInfoPropertyPlaybackRate,
```

Comment:
I agree, but I'll not remove the code from Android, as all calls to `updatePlayback` would require the `elapsedTime`. I'll probably leave a note that providing the elapsed time is not only better for the performance but also for iOS compatibility.

Ah, before I forget, for the other Android states, can you set them as aliases to `STATE_PAUSED` just to not break compatibility? Something like this:
```js
STATE_PLAYING: NativeMusicControl.STATE_PLAYING,
STATE_PAUSED: NativeMusicControl.STATE_PAUSED,
STATE_ERROR: NativeMusicControl.STATE_PAUSED,
STATE_STOPPED: NativeMusicControl.STATE_PAUSED,
STATE_BUFFERING: NativeMusicControl.STATE_PAUSED
```

Thanks :)